lib/wasm-contract/src/contract.rs

```diff
@@ -55,13 +55,21 @@ pub enum Import {
 
 // TODO: figure out this separator... '/' is a valid character in names so we can have collisions
 impl Import {
+    pub fn format_fn_key(ns: &str, name: &str) -> String {
+        format!("{}/{}", &ns, &name)
+    }
+
+    pub fn format_global_key(name: &str) -> String {
+        name.to_string()
+    }
+
     /// Get the key used to look this import up in the Contract's import hashmap
     pub fn get_key(&self) -> String {
         match self {
             Import::Func {
                 namespace, name, ..
-            } => format!("{}/{}", &namespace, &name),
-            Import::Global { name, .. } => name.clone(),
+            } => Self::format_fn_key(&namespace, &name),
+            Import::Global { name, .. } => Self::format_global_key(&name),
```

Comment:
Why we are not using the namespace for globals?

https://github.com/WebAssembly/spec/blob/master/test/core/globals.wast#L302

Reply:
It was missing from the design doc we made!

Updated now